website/client/components/achievements/newStuff.vue

```diff
@@ -50,6 +50,7 @@
     methods: {
       tellMeLater () {
         this.$store.dispatch('user:newStuffLater');
```

Comment:
Hmm this route on the server marks user.flags.newStuff as false, so maybe the only thing to do is to make sure it's marked as false on the client too? We can just add it to the user:newStuffLater action here

Reply:
Ah, okay. That makes sense. I'll move it to the action then.